packages/bot-web-ui/src/stores/quick-strategy-store.ts

```diff
@@ -333,7 +329,7 @@ export default class QuickStrategyStore {
 
         this.setSymbolDropdown(symbol_options);
 
-        if (!this.selected_symbol && symbol_options.length) {
+        if (!this.selected_symbol.value && symbol_options.length) {
```

Comment:
why is this check for value required wasn't `this.selected_symbol` right ?

Reply:
> why is this check for value required wasn't `this.selected_symbol` right ?

@rupato-deriv this.selected_symbol is an object. Object to boolean is always true (empty or filled). But we need to know whether we have something in the object or no.